test/integration/scheduler/scheduler_test.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package scheduler
 
 import (
-	"github.com/onsi/ginkgo"
+	ginkgo "github.com/onsi/ginkgo/v2"
```

Comment:
I suppose this is not needed.

Reply:
ups, was needed at the beginning of the reorg , removing....

Reply:
done